pyproject.toml

```diff
@@ -59,6 +59,7 @@ dev = [
     "interrogate==1.5.0",
     "coverage==6.5.0",
     "ruff==0.0.290",
+    "multi-storage-client>=0.12.2",
```

Comment:
Why is this only in the dev optional dependency list? Wouldn't it be better to put it in some storage specific optional dependency list?